ideascube/widgets.py

```diff
@@ -1,6 +1,101 @@
 from django.forms import widgets
 
 from ideascube.configuration import get_config
+from django.utils.html import mark_safe, escape
+from django.utils.encoding import force_text
+from django.forms.utils import flatatt
+
+import bleach
+
+
+_script_template = '''<script type="text/javascript">tinymce.init({{
+selector: "#{widget_id}",
+inline: true,
+theme: "inlite",
+hidden_input: false,
+menubar: false,
+toolbar: false,
+selection_toolbar: {settings[selection_toolbar]},
+insert_toolbar: {settings[insert_toolbar]},
+plugins: {settings[plugins]}{settings[extra_settings]}
+}});</script>'''
+
+
+class RichTextEntry(widgets.Widget):
+    def __init__(self, *,
+            with_media = False,
+            selection_toolbar=None,
```

Comment:
Indeed, maybe for now we should start small and only have the `with_media=True|False` option.

And if we ever feel the need to add more options, we can see at that moment.

Right now we have three fields for which we want the rich text editor:

* the media document summary
* the blog article summary
* the blog article content

The first two are simple, and should only contain text. However allowing blod, italic and simple things like this in those seems appropriate.

On the other hand, we do want media for the last one.

So `with_media` seems like an appropriate knob for now, let's just do this.